src/box/gc.h

```diff
@@ -96,15 +97,29 @@ struct gc_consumer {
 	struct tt_uuid uuid;
 	/** Human-readable name of object. */
 	char object_name[GC_NAME_MAX];
-	/** The vclock tracked by this consumer. */
+	/**
+	 * The vclock tracked by this consumer, is consistent with persistent
+	 * state if the consumer is persistent.
+	 */
 	struct vclock vclock;
+	/** Asynchronously advanced vclock. */
+	struct vclock volatile_vclock;
```

Comment:
Done.
The second vclock was needed because one of them (`volatile_vclock`) was advanced on in-memory update and another one was advanced on commit in `_gc_consumers`. Now we advance consumers only in-memory, so the second consumer is not needed.